packages/cli/configuration/src/generators-yml/generatorInvocations.ts

```diff
@@ -133,5 +133,19 @@ export const GENERATOR_INVOCATIONS: Record<GeneratorName, Omit<GeneratorInvocati
             location: "local-file-system",
             path: "../sdks/csharp"
         }
+    },
+    [GeneratorName.SWIFT_MODEL]: {
+        version: "0.0.0",
+        output: {
+            location: "local-file-system",
+            path: "../sdks/swift/model"
+        }
+    },
```

Comment:
It looks like we omit the `output` configuration for the other _model_ generators above. Let's keep this consistent for now.

Reply:
donezo